File: DigitalContentPlatform.Core/Entities/SharedContent.cs
public class SharedContent
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string Url { get; set; } // ссылка на файл или изображение
    public string ContentType { get; set; } // например: image, video, document
    public DateTime CreatedAt { get; set; }
}

File: digital-content-platform-client/src/types/sharedContent.ts
export interface SharedContentDto {
    id: string;
    title: string;
    description: string;
    url: string;
    contentType: string; // например: "image", "video", "document"
    createdAt: string;
}

export interface CreateSharedContentDto {
    title: string;
    description: string;
    file: File | null;
    contentType: string;
}